### PR TITLE
fix: Agenda : if external visio have a long url, unable to store the event

### PR DESCRIPTION
**Pull request by rdenarie**
Before this fix, if the visio url is longer than 500 char, and when it is used for a space event, the visio link was not savec because the filed in db is limited to 500chars This commit add a check when filling the space meeting url, to not add a link longer than 500 chars

### CODE REVIEW

File: webapp/src/main/webapp/vue-apps/SpaceAdministrationVisioConference/components/VideoConferenceLinkDrawer.vue [hunk 1]
```diff
@@ -127,7 +128,12 @@ export default {
         this.$root.$emit('refresh-video-conferences');
         this.close();
       });
+    },
+    checkUpdate(value) {
+      console.log('update', value);
+      console.log('validForm', this.isValidForm);
```
Comment (ahamdi) [inline webapp/src/main/webapp/vue-apps/SpaceAdministrationVisioConference/components/VideoConferenceLinkDrawer.vue]: IMO this is not needed

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: webapp/src/main/webapp/vue-apps/SpaceAdministrationVisioConference/components/VideoConferenceLinkDrawer.vue (hunk 1 above)
Comment (rdenarie) [reply webapp/src/main/webapp/vue-apps/SpaceAdministrationVisioConference/components/VideoConferenceLinkDrawer.vue]: updated

**Comment by rdenarie** [comment]
@mkrout @azayati @ahamdi up to review, thanks

**Review by azayati — APPROVED** [review]
LGTM

**Comment by rdenarie** [comment]
cc @ahamdi as you made a change request, I need your validation to merge

**Review by ahamdi — APPROVED** [review]

**Pull request merged**